docs/src/examples/sinusoid_example_toml.md

```diff
@@ -0,0 +1,77 @@
+# TOML interface for fitting parameters of a sinusoid
+
+Here we revisit the [simple example](@ref sinusoid-example). 
+
+The purpose of this example is to demonstrate how the EKP tools can interact with models in a non-intrusive fashion. In particular, this example will be useful if the model is written in another language, requires HPC management, or requies additional data processing stages.
+
+After instantiation, this example calibration is called from a simple script-language (bash) by calling
+```
+> bash calibrate_script
+```
+Which calls the following sequence of runs (here in pseudo-code)
```

Comment:
More than pseudocode, the loop syntax is in Julia, right? That can be confusing. Why not use bash syntax? 

The fact that we are running the computer model with `Julia --project` is also quite specific. We can leave that if we comment that this need not be a Julia call (for instance, it could be python run_computer_model.py)

Reply:
I'll give this piece another go, originally I wanted something easy to read but which showed that which files we called where, in the example